work/src/org/kuali/kfs/module/ar/service/impl/AccountsReceivableModuleBillingServiceImpl.java

```diff
@@ -160,6 +164,30 @@ public boolean isContractsGrantsBillingEnhancementActive() {
         return getConfigurationService().getPropertyValueAsBoolean(KFSConstants.CONTRACTS_GRANTS_BILLING_ENABLED);
     }
 
+    public String getDefaultDunningCampaignCode() {
+        String parameterDunningCampaignCode = "";
+        if (getParameterService().parameterExists(DunningCampaign.class, ArConstants.DEFAULT_DUNNING_CAMPAIGN_PARAMETER)) {
```

Comment:
Do we need this anymore?  Can't we just send back the empty string as a default?